pibieta/imbalanced_learning
# E book config

# jupyter-book build
In this PR we include the config files needed for building the v 0.0 of our imbalanced learning book. 
## What has changed?
Basically, two files were added to our root folder:
* `_config.yml`
* `_toc.yml`
Additionally, the base (suggested) environment file `env/ibc.yml` was updated to contain the [jupyter-book](https://github.com/executablebooks/jupyter-book) package.

## How to review this PR?
The reviewer could go through the following steps:
1. update (or create) your environment by installing the [jupyter-book](https://github.com/executablebooks/jupyter-book) package. One option is to use the `env\ibc.yml` file as follows:
```bash
conda activate myenv
conda env update --file env\ibc.yml.yml --prune
```
2. run the following command, make sure you are at your local repository's root folder:
```bash
jupyter-book build .
``` 
3. Check the folder `_build` and look for the `index.html` file. Open that file in a browser and voilà!

Have fun!
